src/curated-corpus/components/ScheduledItemCardWrapper/ScheduledItemCardWrapper.styles.tsx

```diff
@@ -9,6 +9,7 @@ export const useStyles = makeStyles((theme: Theme) =>
       height: '100%',
       display: 'flex',
       flexDirection: 'column',
+      position: 'relative',
     },
```

Comment:
I'm pretty sure this is totally against all possible CSS standards and conventions but it works.